README.md

```diff
@@ -11,77 +11,65 @@
 - STAC extensions used:
   - [classification](https://github.com/stac-extensions/classification/)
   - [datacube](https://github.com/stac-extensions/datacube/)
+  - [item-assets](https://github.com/stac-extensions/item-assets)
   - [processing](https://github.com/stac-extensions/processing/)
-  - [proj](https://github.com/stac-extensions/projection/)
+  - [projection](https://github.com/stac-extensions/projection/)
   - [raster](https://github.com/stac-extensions/raster/)
-  - [version](https://github.com/stac-extensions/version/)
+  - [scientific](https://github.com/stac-extensions/scientific)
 - Extra fields:
-  - None
+  - `esa_cci_lc:version`: Land cover product version.
+  - `esa_cci_lc:tile`: Geographic coordinate of the lower left corner of the COG tile, rounded to the nearest degree, e.g., N45W180.
```

Comment:
Is the `grid` extension usable here?

Reply:
Based on our offline discussion today, I'm removing the grid extension. We are using a custom tiling scheme with a user defined tile size, so not a compelling use case. Incidentally, I am kicking the tires on the the grid extension on LCMAP, which follows the Landsat ARD tile grid.